src/components/vm/nics/nicBody.tsx

```diff
@@ -271,3 +290,311 @@ export const NetworkTypeAndSourceRow = ({
         </>
     );
 };
+
+export function portForwardText(pf: VMInterfacePortForward): string {
+    let text = "";
+
+    if (pf.address)
+        text += pf.address;
+    if (pf.dev)
+        text += pf.dev;
+    if (!pf.range.some(r => r.exclude != "yes")) {
+        if (text != "")
+            text += ", ";
+        text += _("all ports");
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2331-3b8315e1-20251022-060022-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)